next.config.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
         hostname: '**',
       },
     ],
+    deviceSizes: [640, 750, 760, 828, 928, 1080, 1200, 1432, 1592, 1920, 2048, 3840],
+    imageSizes: [424],
```

Comment:
Can you elaborate more regarding the purpose of increasing the number of device sizes from a default one, as well as decreasing the number of default image sizes?

Reply:
I added the sizes for testing, already removed them